docs/workflow_tasks/discover_granules.md

```diff
@@ -48,6 +48,10 @@ A list of buckets with types that will be used to assign bucket targets based on
 
 A Cumulus [collection](https://github.com/nasa/cumulus/blob/master/packages/api/lib/schemas.js) object.    Used to define granule file groupings and granule metadata for discovered files.   The collection object utilizes the collection type key to generate types in the output object on discovery.
 
+##### Collection Meta
+
+If the collection is configured with `collection.meta.allFilesPresent` set to `true`, the task will remove granules's missing files. Otherwise, the default behavior ignores missing files in granules.
```

Comment:
```suggestion
If the collection is configured with `collection.meta.allFilesPresent` set to `true`, the task will remove granules with all the files. Otherwise, by default, task continue to process granules without all the files.
```
`remove granules's missing files` is confusing

Reply:
The wording above is still confusing. Do you mean something like the following?

> If the collection is configured with `collection.meta.allFilesPresent` set to `true`, the task will include _only_ granules where all files specified in the `collection.files` list are present, thus excluding granules where at least one file is absent. Otherwise, by default (i.e., `collection.meta.allFilesPresent` defaults to `false`), the task includes _every_ granule, whether or not all of its files are present.